indico/modules/events/abstracts/templates/_abstract.html

```diff
@@ -0,0 +1,201 @@
+                                    Speaker: {{ names }}
+                                {% pluralize %}
+                                    Speakers: {{ names }}
+                                {% endtrans %}
+                            </div>
+                        {% endif %}
+                        {# TODO custom fields #}
+                        {# TODO comment #}
+                    </div>
+                </div>
+            </div>
+
+            {% if abstract.state.name != 'submitted' %}
+                <div class="abstract-public-comment">
+                    <div class="abstract-public-comment-pic"></div>
+                    <div class="abstract-public-comment-box">
+                        <div class="i-box-header">
+                            {% trans name=abstract.judge.name, action=abstract.state.title|lower, date=abstract.judgment_dt|format_human_date %}
+                                <strong>{{ name }}</strong> {{ action }} this abstract on {{ date }}.
+                            {% endtrans %}
+                        </div>
+                        <div class="i-box-content">
+                            {% if abstract.state.name == 'accepted' %}
+                                {% if abstract.accepted_contrib_type and abstract.accepted_track %}
+                                    {% trans type=abstract.accepted_contrib_type.name, track=abstract.accepted_track.title %}
+                                        The abstract was accepted
+                                        as <span class="abstract-contribution-type">{{ type }}</span>
+                                        for track <span class="abstract-track">{{ track }}</span>
+                                    {% endtrans %}
+                                {% elif abstract.accepted_contrib_type %}
+                                    {% trans type=abstract.accepted_contrib_type.name %}
+                                        The abstract was accepted
+                                        as <span class="abstract-contribution-type">{{ type }}</span>
+                                    {% endtrans %}
+                                {% elif abstract.accepted_track %}
+                                    {% trans track=abstract.accepted_track.title %}
+                                        The abstract was accepted
+                                        for track <span class="abstract-track">{{ track }}</span>
+                                    {% endtrans %}
+                                {% else %}
+                                    {% trans %}
+                                        The abstract was accepted
+                                    {% endtrans %}
+                                {% endif %}
+                                {% trans url='#' %}
+                                    and a <a href="{{ url }}">contribution</a> was created in the event.
```

Comment:
Each sentence on each side of `and` can be seen as a standalone sentence, but okay, I'll repeat it four times.